README.md

```diff
@@ -37,11 +41,41 @@ To load/activate the Python Virtual Environment in Windows 10 CMD use the follow
 .venv/Scripts/activate.bat
 ```
 
+To load/activate the Python Virtual Environment in Mac OS or Ubuntu 22.04 LTS use the following command: 
+```
+source .venv/bin/activate
+```
+
 ### Install the Python Required Packages
 
 To install the required Python packages inside the loaded/activated virtual enviroment run the following command:
 ```
 python -m pip install -r requirements.txt
 ```
 
-## How to Build and Run
+### How to install Django on cmd or Terminal
+for Windows: 
+```
+pip install django  
+```
+for MacOS
+```
+pip3 install django  
+```
+
+## Check django version: 
+```
+python -m django –version (Windows)
+```
+
+```
+python -m django –version  (MacOS)
+```
+
```

Comment:
Remove how to install Django from the CLI from the README, because Django is specified in the requirements.txt file